<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WRMRegItem</class>
 <widget class="QWidget" name="WRMRegItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>35</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reg_addr">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>5</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>寄存器地址</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_no">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_off">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
